src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,157 @@
+footer {
+  font-size: 0.875rem;
+  background: $gray-dark;
+  color: transparentize($white,0.5);
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.footer-links {
+  text-align: center;
+  list-style-type: none;
+  padding-left: 0;
+  margin-bottom: 0;
+
```

Comment:
Also @mmmavis this is a great guide if you're not already aware of it:

https://css-tricks.com/snippets/css/a-guide-to-flexbox